sfdx/README.md

```diff
@@ -38,19 +38,19 @@ Help topics, type "sfdx help TOPIC" for more details:
 -> sfdx mobilesdk:ios --help
 Usage: sfdx ios:COMMAND [command-specific-options]
 
-Commands for ios, type "sfdx help ios:COMMAND" for more details:
+Commands for the ios option. Type "sfdx help ios:COMMAND" for more details.
 
- sfdx mobilesdk:ios:create           # create ios native or native_swift mobile application
- sfdx mobilesdk:ios:createWithTemplate # create ios native or native_swift mobile application from a template
- sfdx mobilesdk:ios:version          # print version of Mobile SDK
+ sfdx mobilesdk:ios:create           # Create an iOS native or native_swift mobile application
+ sfdx mobilesdk:ios:createWithTemplate # Create an iOS native or native_swift mobile application from a template
+ sfdx mobilesdk:ios:version          # Print version of Mobile SDK
 ```
 
 ### Create Objective-C (native) or Swift (native_swift) application
 ```
 -> sfdx mobilesdk:ios:create --help
 Usage: sfdx mobilesdk:ios:create
 
-create ios native or native_swift mobile application
+Create an iOS native or native_swift mobile application.
```

Comment:
That's the same text shown when doing `sfdx mobilesdk:ios --help` so it will be lower case if we want to follow the force plugin example.
Also that line is generated from the topic (the xx in mobilesdk:xx) and we don't want them to have to type mobilesdk:iOS, do we?

Reply:
Hmm. If the reader can't infer that "ios" the command line token is a stand-in for "iOS" the trademark that's used in a well-formed English sentence, well...not much we can do for them. The correct camelcase also helps the user see that, yes, we are talking about the mobile operating system rather than some beautiful island off the coast of Greece. Again, the period at the end is optional.